tests/integration/test_xrdsst.py

```diff
@@ -1012,6 +1012,15 @@ def step_client_unregister(self):
                     assert len(found_client) > 0
                     assert found_client[0]["status"] == ClientStatus.DELETION_IN_PROGRESS
 
+    def step_make_owner(self):
+        member = 'DEV:COM:222'
+        with XRDSSTTest() as app:
+            client_controller = ClientController()
+            client_controller.app = app
+            configuration = client_controller.create_api_config(self.config["security_server"][0], self.config)
```

Comment:
It seems you are performing the operation for all security servers, but creating the api key only for the first one, it will probably not work that way